### PR TITLE
[FIX] project_{stock,mrp,purchase}: add helpers to top bar actions

### PR DESCRIPTION
**Pull request by deneuvillem**
Add helpers for the following actions:
- Bills of Materials
- To WH
- From WH
- Stock Moves
- Purchase Orders

task-4210573
version-18.0

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project_purchase/models/project_project.py [hunk 1]
```diff
@@ -36,6 +36,11 @@ def action_open_project_purchase_orders(self):
             'context': {
                 'default_project_id': self.id,
             },
+            'help': "<p class='o_view_nocontent_smiling_face'>%s</p><p>%s</p>" % (
+                _("No purchase order found. Let's create one."),
+                _("Once you ordered your products to your supplier, confirm your request for quotation and it will turn \
```
Comment (damrOdoo) [inline addons/project_purchase/models/project_project.py:41]: I think the correct sentence is 'Once you ordered your products _from_ your supplier, [...]'

**Review by damrOdoo — COMMENTED** [review]
Thanks for your work. I just left one small comment.
edit: a bit soon, I didn't see the task was not greened yet in testing india :sweat_smile:

File: addons/project_purchase/models/project_project.py (hunk 1 above)
Comment (deneuvillem) [reply addons/project_purchase/models/project_project.py]: Indeed, nice catch! :muscle:

**Comment by damrOdoo** [comment]
Lgtm. Thanks for your fix.

**Review by xavierbol — COMMENTED** [review]
Thanks for your work. 🙂 

I left 2 comments. 🙂 

Can you also change the commit tag since it is more a fix? 🙂 

Thanks in advance!

**Review by Whenrow — APPROVED** [review]

**Review by xavierbol — APPROVED** [review]
LGTM, thanks again for your work! 🙂 

robodoo r+

**Pull request closed without merging**